content/html/concepts/tags/terms/param/param.md

```diff
@@ -0,0 +1,36 @@
+---
+Title: '<param>'
+Description: 'Used to pass parameters to a resource defined in an object tag.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Tags'
+  - 'Parameters'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<param>` tag is used within an [`<object>`](https://www.codecademy.com/resources/docs/html/tags/object) tag to pass parameters to the resource it is displaying.
+
+## Syntax
+
+```html
+<param name="parameter name" value="parameter value" />
+```
+
+The `<param>` tag uses two attributes:
+
+- A `name` for the parameter.
+- A `value` to specify the named parameter's value.
```

Comment:
It would be nice to get some other examples in here on what the name parameter can be set to other than the one described in the example.

Reply:
I noticed a lot of other sources all say the same thing without giving extra examples other than autoplay or controller. Are these the two main examples?

Reply:
@yangc95, The other significant one is passing parameters to a Java applet, replacing some of the attributes of the depreciated applet tag. That seemed a little involved for an example.

Reply:
Alright, pass on this then~